src/commands/commerce/init.js

```diff
@@ -42,66 +44,79 @@ export class InitCommand extends Command {
-    console.log(`${boldWhite}Preview your storefront:${reset} https://main--${githubRepo}--${githubOrg}.aem.page/`)
-    console.log(`${boldWhite}Run your storefront locally:${reset} "aio commerce:dev"`)
-    if (meshUrl) {
-      console.log(`${boldWhite}Try out your API:${reset} ${meshUrl}`)
-      console.log(`To check the status of your Mesh, run ${boldWhite}aio api-mesh status${reset}`)
-      console.log(`To update your Mesh, run ${boldWhite}aio api-mesh update mesh_config.json${reset}`)
-      if (meshDetailsPageURL) {
-        meshDetailsPageURL && console.log(`${boldWhite}View your Mesh details:${reset} ${meshDetailsPageURL}`)
+      console.log('\n************************************************')
+      console.log(`🎉 ${boldWhite}Setup complete!${reset} 🎉\n`)
+      console.log(`${boldWhite}Customize your code:${reset} https://github.com/${githubOrg}/${githubRepo}`)
+      console.log(`${boldWhite}Edit your content:${reset} https://da.live/#/${githubOrg}/${githubRepo}`)
+      console.log(`${boldWhite}Manage your config:${reset} https://da.live/sheet#/${githubOrg}/${githubRepo}/configs-stage`)
+      console.log(`${boldWhite}Preview your storefront:${reset} https://main--${githubRepo}--${githubOrg}.aem.page/`)
+      console.log(`${boldWhite}Run your storefront locally:${reset} "aio commerce:dev"`)
+      if (meshUrl) {
+        console.log(`${boldWhite}Try out your API:${reset} ${meshUrl}`)
+        console.log(`To check the status of your Mesh, run ${boldWhite}aio api-mesh status${reset}`)
+        console.log(`To update your Mesh, run ${boldWhite}aio api-mesh update mesh_config.json${reset}`)
+        if (meshDetailsPageURL) {
+          meshDetailsPageURL && console.log(`${boldWhite}View your Mesh details:${reset} ${meshDetailsPageURL}`)
+        }
       }
-    }
-    console.log('For next steps, including how to customize your storefront and make it your own, check out our docs:\nhttps://experienceleague.adobe.com/developer/commerce/storefront/')
+      console.log('For next steps, including how to customize your storefront and make it your own, check out our docs:\nhttps://experienceleague.adobe.com/developer/commerce/storefront/')
+      console.log('************************************************\n')
+      // cleanup
+      config.delete('commerce')
+      // reset github org and repo, for aio commerce:dev command
+      config.set('commerce.github', {
+        org: githubOrg,
+        repo: githubRepo
+      })
+    } catch (error) {
+      console.log('\n************************************************')
+      console.error('❌ Sorry to see the setup fail, please run the following commands before attempting a retry:\n')
+      console.log(`Delete Github repo. Run ${boldWhite}gh repo delete ${githubOrg}/${githubRepo}${reset}`)
+      console.log(`Delete API Mesh. Run ${boldWhite}aio api-mesh delete${reset}`)
+      console.log(`Delete the Content at ${boldWhite}https://da.live/#/${githubOrg}/${githubRepo}${reset}`)
```

Comment:
You know better about retry logic for non mesh stuff. I added the steps I usually follow. If I missed something let me know.

Reply:
The repo delete command will not work because the token used during lab does not have this permission. we _could_ add it, but I'm hesitant to just give them ability to call this function and potentially delete the wrong repo :)

I think for lab, it is fine to leave this, and we will just instruct them to retry with a new repo name (like `seat-10-1`).

The repo/da folder name does not really matter. What matters is that they do not reuse an existing one :)